GsaGH/Components/5_Results/ReactionForces.cs

```diff
@@ -226,13 +214,11 @@ List<GsaResultsValues> vals
       da.SetDataTree(6, outRotZ);
       da.SetDataTree(7, outRotXyz);
       da.SetDataTree(8, outIDs);
-
-      PostHog.Result(result.CaseType, 0, GsaResultsValues.ResultType.Force);
     }
 
     protected override void UpdateUIFromSelectedItems() {
-      _forceUnit = (ForceUnit)UnitsHelper.Parse(typeof(ForceUnit), _selectedItems[0]);
-      _momentUnit = (MomentUnit)UnitsHelper.Parse(typeof(MomentUnit), _selectedItems[1]);
+      _forceUnit = (ForceUnit)UnitsHelper.Parse(typeof(ForceUnit), _selectedItems[1]);
+      _momentUnit = (MomentUnit)UnitsHelper.Parse(typeof(MomentUnit), _selectedItems[2]);
```

Comment:
You need to handle serialising existing components, see https://github.com/arup-group/GSA-Grasshopper/blob/epic/GsaGH-366/GsaGH/Components/5_Results/BeamForcesAndMoments.cs#L231